lib/build.gradle

```diff
@@ -68,6 +68,7 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'org.threeten:threetenbp:1.6.4'
```

Comment:
what is this for?

Reply:
oo is it cuz of the java.time issue you told me?

Reply:
Yes, that was previously only used from the example app before, but expiration date validation required it also in the lib module.